addons/imjp94.yafsm/scenes/flowchart/FlowChartGrid.gd

```diff
@@ -7,24 +7,27 @@ func _ready():
     queue_redraw()
 
 func _draw():
-    self.position = flowchart.position
-    self.size = flowchart.size*100  # good with min_zoom = 0.5 e max_zoom = 2.0
-
     # Original Draw in FlowChart.gd inspired by:
 	# https://github.com/godotengine/godot/blob/6019dab0b45e1291e556e6d9e01b625b5076cc3c/scene/gui/graph_edit.cpp#L442
+    
+    self.position = flowchart.position
+    self.size = flowchart.size*100  # Extents of the grid. good with min_zoom = 0.5 and max_zoom = 2.0
 
     var zoom = flowchart.zoom
     var snap = flowchart.snap
 
-    var offset = -Vector2(1, 1)*10000  # good with min_zoom = 0.5 e max_zoom = 2.0
+    var offset = -Vector2(1, 1)*1000  # Origin of the grid. Good with min_zoom = 0.5 and max_zoom = 2.0
```

Comment:
Does it impact the performance? Otherwise, it seems that the bigger the number, the better it is.
If it works fine with "10000", it would be better to remove unnecessary changes, to avoid any confusion in case of debugging code with "git blame" in the future